draft-ietf-add-dnr.xml

```diff
@@ -871,43 +870,32 @@
         <t>IANA is requested to assign the following new DHCPv6 Option Code in
         the registry maintained in <xref target="DHCPV6"></xref>.</t>
 
-        <texttable>
-          <ttcol>Value</ttcol>
+        <t><figure>
+            <artwork align="center"><![CDATA[+=======+===============+============+===========+================+
+| Value | Description   | Client ORO | Singleton | Reference      |
+|       |               |            | Option    |                |
++=======+===============+============+===========+================+
+| TBA1  | OPTION_V6_DNR | Yes        | No        | [ThisDocument] |
++-------+---------------+------------+-----------+----------------+
 
```

Comment:
This is the opposite of what I was looking to have happen.  The `<texttable>` tables are good.  I was hoping that you would change the next section to use a real table, rather than regressing this one.

Reply:
Will fix this.